addons/sourcemod/scripting/include/shop.inc

```diff
@@ -205,13 +205,14 @@ forward bool Shop_OnClientLuckProcess(int client);
 
 /**
  *	Called when a player is nearly to win an item
- *	
- *	@param client			Client index who is nearly to win
- *	@param item_id			Item id of the item
- *	
- *	@return true to allow and false to block
+ *	-
+ *	@param client			client index
+ *	@param item_id			Item id
+ *	@param luck_chance		luck chance (by reference)
+ *	-
+ *	@return Plugin_Changed to apply custom changes. >= Plugin_Handled to block, Plugin_Continue - to proceed without changes.
 */
-forward bool Shop_OnClientShouldLuckItem(int client, ItemId item_id);
+forward Action Shop_OnClientShouldLuckItem(int client, ItemId item_id, int &luck_chance);
```

Comment:
This is very breaking change, because `false` now works as `true` (`Plugin_Continue == 0`).
Maybe deprecate this forward and create another one? Or we support breaking changes like this?

Reply:
We can rename that native to `Shop_OnClientShouldLuckItemChance` and make old one to be deprecated, without any functionality on it.